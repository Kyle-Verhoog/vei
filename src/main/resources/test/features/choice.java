// FAIL
// Java 1.3
public class Choice {
  public int m(int x) {
    return x==42 ? 87 : x;
  }
}