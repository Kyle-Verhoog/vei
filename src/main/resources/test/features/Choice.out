PUBLIC ┃ public
CLASS ┃ class
IDENTIFIER ┃ A
{ ┃ {
PUBLIC ┃ public
INT ┃ int
IDENTIFIER ┃ m
( ┃ (
INT ┃ int
IDENTIFIER ┃ x
) ┃ )
{ ┃ {
RETURN ┃ return
IDENTIFIER ┃ x
== ┃ ==
INTEGER_LITERAL ┃ 42
? ┃ ?
INTEGER_LITERAL ┃ 87
: ┃ :
IDENTIFIER ┃ x
; ┃ ;
} ┃ }
} ┃ }